### ISSUE TITLE
Error on recombine ebsynth

### ISSUE THREAD

**Issue by orfeaus** [CiaraStrawberry/TemporalKit#42]
Running Stable Diffusion (5ab7f21) on an M2 Mac (Venture 13.3.1).

Getting the following error when hitting the recombine button after successfully preparing ebsynth in the previous step:

```
saving at frame 172
saved to /Users/me/Documents/promo video/TemporalKit/keys/keys00172.png at size (1820, 1024)
[2, 7, 12, 17, 22, 27, 32, 37, 42, 47, 52, 57, 62, 67, 72, 77, 82, 87, 92, 97, 102, 107, 112, 117, 122, 127, 132, 137, 142, 147, 152, 157, 162, 167, 172]
Traceback (most recent call last):
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/routes.py", line 399, in run_predict
    output = await app.get_blocks().process_api(
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/blocks.py", line 1299, in process_api
    result = await self.call_function(
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/venv/lib/python3.10/site-packages/gradio/blocks.py", line 1022, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/venv/lib/python3.10/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/venv/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/venv/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/extensions/TemporalKit/scripts/sd-TemporalKit-UI.py", line 182, in recombine_ebsynth
    return ebsynth.crossfade_folder_of_folders(input_folder,fps=fps,return_generated_video_path=True)
  File "/Users/me/Documents/stable-diffusion/stable-diffusion-webui/extensions/TemporalKit/scripts/Ebsynth_Processing.py", line 134, in crossfade_folder_of_folders
    current_dir = os.path.join(root_folder, dirs[0])
IndexError: list index out of range
```

**Comment by orfeaus** [comment]
I was able to resolve this issue. Turns out the files were not generated correctly in the previous step so when this script went to run, it didn't find the files.

After redoing the previous step in EBsynth and generating the correct files, I was able to perform this last step successfully.

Perhaps this issue can track an enhancement request to provide a clearer error message to the user instructing them to confirm the correct file / folder outputs from the previous step?
